src/views/Alteration.vue

```diff
@@ -49,6 +47,10 @@
       />
 
       <no-fee-summary class="mt-10" />
+      <documents-delivery class="mt-10" />
+
+      <certify-section class="mt-10" />
```

Comment:
What about passing `pleaseValidate` for high level validation (after user clicks File and Pay), and displaying the error styles? (see AlterationSummary.vue)

(The mockups embedded in your ticket don't show this, but see the invision linked in the design ticket: https://projects.invisionapp.com/share/6EYT1CDGYVN#/screens/445482334)

Reply:
(click right arrow to see more slides)

Reply:
added please validate